prototype_source/prototype_index.rst

```diff
@@ -82,6 +82,13 @@ Prototype features are not available as part of binary distributions like PyPI o
    :link: ../prototype/pt2e_quant_ptq_static.html
    :tags: Quantization          
 
+.. customcarditem::
+   :header: TorchInductor with X86 CPU as backend of Quantization in PyTorch 2.0 Export
```

Comment:
maybe "PYTORCH 2.0 EXPORT POST TRAINING STATIC QUANTIZATION with X86 Backend"

Reply:
Thanks for the suggestions and changed.